checkov/github/dal.py

```diff
@@ -13,25 +14,21 @@ class Github(BaseVCSDAL):
     def __init__(self) -> None:
         super().__init__()
         self.github_conf_dir_path: str
-        self.github_org_security_file_path: str
-        self.github_branch_protection_rules_file_path: str
-        self.github_org_webhooks_file_path: str
-        self.github_repository_webhooks_file_path: str
-        self.github_repository_collaborators_file_path: str
+        self.github_conf_file_paths: dict[str, Path]
 
     def setup_conf_dir(self) -> None:
-        # files downloaded from github will be persistent in this directory
         github_conf_dir_name = os.getenv('CKV_GITHUB_CONF_DIR_NAME', 'github_conf')
         self.github_conf_dir_path = os.path.join(os.getcwd(), github_conf_dir_name)
-        self.github_org_security_file_path = os.path.join(self.github_conf_dir_path, "org_security.json")
-        self.github_branch_protection_rules_file_path = os.path.join(self.github_conf_dir_path,
-                                                                     "branch_protection_rules.json")
-        self.github_org_webhooks_file_path = os.path.join(self.github_conf_dir_path,
-                                                          "org_webhooks.json")
-        self.github_repository_webhooks_file_path = os.path.join(self.github_conf_dir_path,
-                                                                 "repository_webhooks.json")
-        self.github_repository_collaborators_file_path = os.path.join(self.github_conf_dir_path,
-                                                                      "repository_collaborators.json")
+        self.github_conf_file_paths = {
+            "org_security": Path(self.github_conf_dir_path) / "org_security.json",
+            "branch_protection_rules": Path(self.github_conf_dir_path) / "branch_protection_rules.json",
+            "org_webhooks": Path(self.github_conf_dir_path) / "org_webhooks.json",
+            "repository_webhooks": Path(self.github_conf_dir_path) / "repository_webhooks.json",
+            "repository_collaborators": Path(self.github_conf_dir_path) / "repository_collaborators.json"
+        }
+        for _github_conf_type, file_path in self.github_conf_file_paths.items():
+            if os.path.isfile(file_path):
+                os.remove(file_path)
```

Comment:
@marynaKK please use `shutil.rmtree` to clean the entire dir

Reply:
I didn't want to remove potentially other files that the user put there. Is it ok to remove the entire folder?

Reply:
fixed!